### PR TITLE
Added the option to access drive properties by right clicking on the drive widget

### LINKED ISSUE #2232: Access Drives properties by right clicking widgets
We should be able to access Drives properties by right clicking their widgets, just like we can currently do with drives in the sidebar.

App Version: v0.19.0
Windows Version: Windows 10 20H2

### PR DESCRIPTION
**Pull request by suvrat29**
Added the ability to open drive properties from the home widgets. This closes #2232

### CODE REVIEW

File: Files/UserControls/Widgets/DrivesWidget.xaml [hunk 1]
```diff
@@ -12,8 +12,8 @@
     xmlns:muxc="using:Microsoft.UI.Xaml.Controls"
     Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
     mc:Ignorable="d">
-    <StackPanel Spacing="12">
-        <TextBlock
+	<StackPanel Spacing="12">
```
Comment (tsvietOK) [inline Files/UserControls/Widgets/DrivesWidget.xaml:15]: Don't change tabulation.

Comment (suvrat29) [reply Files/UserControls/Widgets/DrivesWidget.xaml:15]: Visual studio actually did it automatically. Should I close this and create a pr with changing it from a text editor?

File: Files/UserControls/Widgets/DrivesWidget.xaml.cs [hunk 2]
```diff
@@ -57,6 +72,12 @@ private async void OpenInNewWindow_Click(object sender, RoutedEventArgs e)
             await Interaction.OpenPathInNewWindowAsync(item.Path);
         }
 
+        private async void OpenDriveProperties_Click(object sender, RoutedEventArgs e)
+        {
+            var item = ((MenuFlyoutItem)sender).DataContext as DriveItem;
+            await new Interaction(associatedInstance).OpenPropertiesWindowAsync(item);
```
Comment (tsvietOK) [inline Files/UserControls/Widgets/DrivesWidget.xaml.cs:78]: `AppInstance.InteractionOperations.OpenPropertiesWindowAsync(item);`?

File: Files/UserControls/Widgets/DrivesWidget.xaml (hunk 1 above)
Comment (tsvietOK) [reply Files/UserControls/Widgets/DrivesWidget.xaml:15]: You can change VS configuration to use four spaces as tabulation. No need to open another PR.

File: Files/UserControls/Widgets/DrivesWidget.xaml.cs (hunk 2 above)
Comment (suvrat29) [reply Files/UserControls/Widgets/DrivesWidget.xaml.cs:78]: Will do.

File: Files/UserControls/Widgets/DrivesWidget.xaml (hunk 1 above)
Comment (suvrat29) [reply Files/UserControls/Widgets/DrivesWidget.xaml:15]: Okay. Will make the suggested changes.

File: Files/UserControls/Widgets/DrivesWidget.xaml.cs (hunk 2 above)
Comment (tsvietOK) [reply Files/UserControls/Widgets/DrivesWidget.xaml.cs:78]: Nvm, keep it as it is.

Comment (suvrat29) [reply Files/UserControls/Widgets/DrivesWidget.xaml.cs:78]: Oh, okay. I was just about to test and commit. I'll keep it as it is.

File: Files/UserControls/Widgets/DrivesWidget.xaml (hunk 1 above)
Comment (suvrat29) [reply Files/UserControls/Widgets/DrivesWidget.xaml]: Made the changes to tabulation as suggested. Please let me know if this is alright.

**Comment by suvrat29** [comment]
The last commit is for issue #3195

File: Files/UserControls/Widgets/DrivesWidget.xaml.cs (hunk 2 above)
Comment (tsvietOK) [inline Files/UserControls/Widgets/DrivesWidget.xaml.cs:78]:
```suggestion
            await AppInstance.InteractionOperations.OpenPropertiesWindowAsync(item);
```

**Comment by tsvietOK** [comment]
@suvrat29 Is cfc7970da5549a558653be7f208171e5920e92de it a part of this PR?

**Comment by suvrat29** [comment]
> @suvrat29 Is cfc7970da5549a558653be7f208171e5920e92de it a part of this PR?

No, it is for a separate issue. But it won't let me create a new PR since this was open.

**Comment by tsvietOK** [comment]
You can create separate branch for this change in your repo.

**Comment by suvrat29** [comment]
I was having some trouble with that today(see the above commits today). But I'll try again. Will you be removing the commit from this PR?

**Comment by tsvietOK** [comment]
Can you create new branch with name `drive_icons`?

**Comment by suvrat29** [comment]
Done. Created a new branch with name drive_icons

**Review by tsvietOK — DISMISSED** [review]

File: Files/UserControls/Widgets/DrivesWidget.xaml [hunk 3]
```diff
@@ -108,6 +108,16 @@
                                                 <FontIcon FontFamily="{StaticResource CustomGlyph}" Glyph="&#xF107;" />
                                             </MenuFlyoutItem.Icon>
                                         </MenuFlyoutItem>
+                                        <MenuFlyoutItem
+                                            x:Name="PropertiesItem"
+                                            x:Uid="BaseLayoutContextFlyoutPropertiesFolder"
```
Comment (yaichenbaum) [inline Files/UserControls/Widgets/DrivesWidget.xaml:113]: Can you create a new string called `DrivesWidgetContextFlyoutPropertiesMenuItem`?

**Review by yaichenbaum — CHANGES_REQUESTED** [review]

File: Files/UserControls/Widgets/DrivesWidget.xaml (hunk 3 above)
Comment (suvrat29) [reply Files/UserControls/Widgets/DrivesWidget.xaml:113]: Yes, I'll make the changes. Just one question(I'm fairly new to this) will it be needed to be created in the resource and string files as well?

Comment (suvrat29) [reply Files/UserControls/Widgets/DrivesWidget.xaml]: I have created the new string `DrivesWidgetContextFlyoutPropertiesMenuItem` and I have added a key in the `en-US` resource file as well. Please let me know if anything more needs to be done.

Comment (yaichenbaum) [reply Files/UserControls/Widgets/DrivesWidget.xaml]: @suvrat29 You did it perfectly, thank you!

**Review by yaichenbaum — APPROVED** [review]

File: Files/UserControls/Widgets/DrivesWidget.xaml (hunk 3 above)
Comment (suvrat29) [reply Files/UserControls/Widgets/DrivesWidget.xaml]: Ah, that's great!

**Review by tsvietOK — APPROVED** [review]

**Pull request merged**